assets/scss/modules/hero.scss

```diff
@@ -11,9 +11,25 @@
   }
 }
 
-.strike::before,
-  .strike::after {
-    background-color: $picton-blue;
-    height: 1rem;
-    width: 37.5rem;
-  }
+.line {
+  background-image: url("../img/line.png");
+  margin-left: 12.5rem;
+  min-height: 2px;
+  width: 37.5rem;
+}
+
+.stay-button {
+  background: rgba(0, 0, 0, 0.0);
```

Comment:
Color literals like `rgba(0, 0, 0, 0)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`0.0` should be written without the mantissa as `0`